model/src/main/kotlin/OrtResult.kt

```diff
@@ -464,4 +464,9 @@ data class OrtResult(
                 result = analyzer.result.withScopesResolved()
             )
         )
+
+    /**
+     * Return an object that can be used to navigate the dependency information contained in this result.
+     */
+    fun dependencyNavigator(): DependencyNavigator = DependencyTreeNavigator(this)
```

Comment:
This could be a lazy property instead to make the call not require parenthesis and since you change it to reuse the same object in a later commit anyway.

Reply:
Done